app/templates/organization/settings.hbs

```diff
@@ -0,0 +1,113 @@
+          Insights
+        </div>
+        <p class="description" data-test-insights-org-settings-description>
+          Make more informed decisions about your development workflow using your build Insights.<br />
+          {{#link-to 'owner.repositories' organization.login (query-params tab='insights')}}View {{organization.name}}'s Insights{{/link-to}}
+        </p>
+        <ul class='travis-radio-list'>
+          <li class='travis-radio-list__item'>
+            <label for="organization_insight_admins">Only allow organization owners to see insights from your private builds</label>
+            <input
+              type="radio"
+              name="organization_insight_vis"
+              class="travis-radio-input"
+              id="organization_insight_admins"
+              value="admins"
+              checked={{eq privateInsightsVisibility 'admins'}}
+              onchange={{action (mut privateInsightsVisibility) 'admins'}}
+            />
+            <label for="organization_insight_admins" class="travis-radio-label"></label>
+          </li>
+          <li class='travis-radio-list__item'>
+            <label for="organization_insight_members">Only allow organization members to see insights from your private builds</label>
+            <input
+              type="radio"
+              name="organization_insight_vis"
+              class="travis-radio-input"
+              id="organization_insight_members"
+              value="members"
+              checked={{eq privateInsightsVisibility 'members'}}
+              onchange={{action (mut privateInsightsVisibility) 'members'}}
+            />
+            <label for="organization_insight_members" class="travis-radio-label"></label>
+          </li>
+          <li class='travis-radio-list__item'>
+            <label for="organization_insight_public">Allow everyone to see insights from your private builds</label>
+            <input
+              type="radio"
+              name="organization_insight_vis"
+              class="travis-radio-input"
+              id="organization_insight_public"
+              value="public"
+              checked={{eq privateInsightsVisibility 'public'}}
+              onchange={{action (mut privateInsightsVisibility) 'public'}}
+            />
+            <label for="organization_insight_public" class="travis-radio-label"></label>
+          </li>
+          <li class='travis-radio-list__item travis-radio-list__item--save'>
+            <button
+              class="settings-button"
+              {{action 'toggleInsightsVisibilityModal'}}
+              disabled={{eq visibilityChange 0}}
+            >Save</button>
+          </li>
+        </ul>
+      </div>
+    </section>
+  {{/if}}
+</section>
+
+{{#if (and features.proVersion isShowingInsightsVisibilityModal)}}
+  {{#modal-dialog
+    toggleInsightsVisibilityModal=(action 'toggleInsightsVisibilityModal')
+    targetAttachment="center"
+    translucentOverlay=true
+  }}
```

Comment:
I’m getting the “should this be extracted” itch here again… but I’m just not sure about it! What do you think?

Reply:
I've gone back and forth on it. It _does_ seem silly to have 2 of almost the exact same thing. At this point, I'm leaning toward trying to do some sort of extraction, though I haven't decided where to draw the lines. Maybe like the radio button list, save button, and modal together make sense as a component. Then at the very least some of the duplication that is in the templates can be reduced.

Reply:
I did end up extracting out a radio button list component and it is helping to keep things a bit more DRY